addons/l10n_sa_edi/wizard/account_move_send.py

```diff
@@ -0,0 +1,52 @@
+from odoo import api, models, fields, _
+
+
+class AccountMoveSend(models.TransientModel):
+    _inherit = 'account.move.send'
+
+    l10n_sa_edi_enable_zatca = fields.Boolean(compute='_compute_l10n_sa_edi_enable_zatca', string="ZATCA")
+
+    @api.depends('move_ids')
+    def _compute_l10n_sa_edi_enable_zatca(self):
```

Comment:
I would appreciate a comment, either here or in the xml, explaining that in a lot of cases, it will be false (as already sent to the gov on post).
It would make it so devs would know that it not being visible is not a bug